stratum/hal/lib/phal/tai/taish_wrapper.cc

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright 2020-present Open Networking Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "stratum/hal/lib/phal/tai/taish_wrapper.h"
+
+#include <algorithm>
+#include <string>
+
+#include "gflags/gflags.h"
+#include "grpcpp/grpcpp.h"
+#include "stratum/lib/macros.h"
+
+DEFINE_string(taish_addr, "", "The gRPC address of TAI shell.");
```

Comment:
Taish server listens on `0.0.0.0:50051` by default. Maybe make the default here `localhost:51000`?